.github/workflows/ci.yml

```
@@ -21,3 +21,23 @@ jobs:
           coverage: xdebug
       - run: composer install
       - run: vendor/bin/phpunit --coverage-text
+
+  PHPStan:
+    name: PHPStan
+    runs-on: ubuntu-20.04
```

Might as well start using the latest Ubuntu version, no?

```suggestion
    runs-on: ubuntu-22.04
```